contracts/L2/token/LivepeerToken.sol

```diff
@@ -0,0 +1,45 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+import {ERC20, ERC20Permit} from "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
+import {ILivepeerToken} from "./ILivepeerToken.sol";
+
+contract LivepeerToken is ILivepeerToken, AccessControl, ERC20Permit {
+    bytes32 public constant MINT_CONTROLLER = keccak256("MINT_CONTROLLER");
+
+    constructor() ERC20("Livepeer Token", "LPT") ERC20Permit("Livepeer Token") {
+        _setupRole(DEFAULT_ADMIN_ROLE, _msgSender());
```

Comment:
Looks like _setupRole is deprecated in favor of _grantRole?

See https://github.com/OpenZeppelin/openzeppelin-contracts/blob/57630d2a6466dff65aa7ca67b3fa23d5e6d1474a/contracts/access/AccessControl.sol#L183

Reply:
Interesting, this might be from an unreleased version. 

As of the latest npm release 4.3.2, `_setupRole` has not been deprecated and `_grantRole` is a private function. hence, there is no other way to setup initial roles.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.3/contracts/access/AccessControl.sol#L182

I guess we have to keep this as is for now and then update it once v4.4 is released

Reply:
I'm okay with moving forward as is and creating an issue to maybe track upgrading although I'm not entirely sure how valuable this will be for minor version bumps.

Reply:
@kautukkundan Got it. Yeah I see that the latest release for OZ is 4.3.2. Let's stick with `_setupRole()` for now then.